doc/sphinx-guides/source/developers/making-releases.rst

```diff
@@ -110,13 +136,40 @@ ssh into the dataverse-internal server and download the release candidate war fi
 
 Go to /doc/release-notes, open the release-notes.md file for the release we're working on, and perform all the steps under "Upgrade Instructions". Note that for regular releases, we haven't bumped the version yet so you won't be able to follow the steps exactly. (For hotfix releases, the version will be bumped already.)
 
+Deploy Release Candidate to QA
+------------------------------
+
+|dedicated|
+
+Deploy the same war file to https://qa.dataverse.org using the same upgrade instructions as above.
+
 Deploy Release Candidate to Demo
 --------------------------------
 
 |dedicated|
 
 Deploy the same war file to https://demo.dataverse.org using the same upgrade instructions as above.
 
+Solicit Feedback from Curation Team
+-----------------------------------
+
+Ask the curation team to test on https://qa.dataverse.org and give them five days to provide feedback.
+
```

Comment:
I considered adding a step here to basically say to continue working on the stuff mentioned above such as:

- regression testing
- performance testing
- finishing up release notes

Basically, we have five days to do stuff like this while we wait for feedback from the curation team.

I did add (below) a step to test building the guides to make sure Jenkins doesn't need a Sphinx upgrade or whatever. This slowed us down last time and it's something we can check during the five day testing period.